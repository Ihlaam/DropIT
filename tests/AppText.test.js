import React from 'react';
import renderer from 'react-test-renderer';
import AppText from '../app/components/AppText';

it('displays text', () => {
    const tree = renderer
      .create(<AppText></AppText>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });