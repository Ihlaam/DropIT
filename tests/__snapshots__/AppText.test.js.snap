// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays text 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#023057",
        "fontFamily": "Avenir",
        "fontSize": 18,
      },
      undefined,
    ]
  }
/>
`;
